pkgs/applications/version-management/gg/default.nix

```diff
@@ -30,7 +30,6 @@ in buildGoModule {
   ldflags = [
     "-s" "-w"
     "-X" "main.versionInfo=${version}"
-    "-X" "main.buildCommit=${commit}"
```

Comment:
Not sure why this line is being removed? Revert?

Reply:
I must have messed something up, fixed it.